client/blocks/reader-full-post/index.jsx

```diff
@@ -764,9 +762,6 @@ export class FullPostView extends Component {
 							) }
 
 							{ post.use_excerpt && <PostExcerptLink siteName={ siteName } postUrl={ post.URL } /> }
-							{ isDailyPostChallengeOrPrompt( post ) && (
-								<DailyPostButton post={ post } site={ site } />
-							) }
```

Comment:
Daily posts no longer function. There is no way you can make this button show for the user because there are no public posts that would trigger it. Whatever we wanted to do with this button, we never did it, so it's a good opportunity for cleanup.